doc/ch-run_desc.rst

```diff
@@ -100,31 +109,30 @@ In addition to any directories specified by the user with :code:`--bind`,
 in as well.
 
 The following host files and directories are bind-mounted at the same location
-in the container. These cannot be disabled.
+in the container. These cannot be disabled. Required (must exist both on host
+and within the image):
 
   * :code:`/dev`
-  * :code:`/etc/passwd`
-  * :code:`/etc/group`
-  * :code:`/etc/hosts`
-  * :code:`/etc/resolv.conf`
   * :code:`/proc`
   * :code:`/sys`
 
-Three additional bind mounts can be disabled by the user:
+Optional (only if path exists on both host and within the image; no error or
+warning if not):
 
-  * Your home directory (i.e., :code:`$HOME`) is mounted at guest
-    :code:`/home/$USER` by default. This is accomplished by mounting a new
-    :code:`tmpfs` at :code:`/home`, which hides any image content under that
-    path. If :code:`--no-home` is specified, neither of these things happens
-    and the image's :code:`/home` is exposed unaltered.
+  * :code:`/etc/hosts`
+  * :code:`/etc/machine-id`
+  * :code:`/etc/resolv.conf`
+  * :code:`/var/lib/hugetlbfs` at guest :code:`/var/opt/cray/hugetlbfs`
+  * :code:`/var/opt/cray/alps/spool`
+  * :code:`$PREFIX/bin/ch-ssh` at guest :code:`/usr/bin/ch-ssh`
```

Comment:
Do we already document why we mount these files/directories? If not, I think that would be helpful for some of the less obvious ones.

I'm thinking these could use some explanation:
 * `/etc/machine-id`
 * `/var/lib/hugetlbfs`
 * `/var/opt/cray/alps/spool`
 * `$PREFIX/bin/ch-ssh`
 
 Thoughts?

Reply:
(1) No (2) I agree